orchagent/pfc_detect_broadcom.lua

```diff
@@ -67,11 +67,10 @@ for i = n, 1, -1 do
                     pfc_on2off_last = tonumber(pfc_on2off_last)
 
                     -- Check actual condition of queue being in PFC storm
-                    if (occupancy_bytes > 0 and packets - packets_last == 0 and pfc_rx_packets - pfc_rx_packets_last > 0) or
+                    if (pfc_rx_packets - pfc_rx_packets_last > 0 and pfc_on2off - pfc_on2off_last == 0 and queue_pause_status_last == 'true' and queue_pause_status == 'true') or
                         -- DEBUG CODE START. Uncomment to enable
```

Comment:
It would be safer to remove this comments and the one 2 lines below. They are no longer safe to be here to comment out the change after this change.